src/Propel/Generator/Platform/MysqlPlatform.php

```diff
@@ -717,10 +718,96 @@ public function getAddColumnDDL(Column $column)
         );
     }
 
+    public function getModifyTableDDL(TableDiff $tableDiff)
+    {
+        $alterTableStatements = '';
+
+        $toTable = $tableDiff->getToTable();
```

Comment:
Sure. I think I did it this way to keep it similar to [the DefaultPlatform version](https://github.com/propelorm/Propel2/blob/master/src/Propel/Generator/Platform/DefaultPlatform.php#L765), but I've no real preference either way; I'll make the change tonight when I'm home from work.

(Alternatively, feel free to merge and make the change yourself.)